test/test_dataloader.py

```diff
@@ -0,0 +1,47 @@
+import unittest
+import torch
+import torch.nn as nn
+from torch.utils.data import Dataset, DataLoader
+import torchaudio
+import math
+import os
+
+
+class TORCHAUDIODS(Dataset):
+
+    test_dirpath = os.path.dirname(os.path.realpath(__file__))
+
+    def __init__(self):
+        self.asset_dirpath = os.path.join(self.test_dirpath, "assets")
+        self.data = [os.path.join(self.asset_dirpath, fn) for fn in os.listdir(self.asset_dirpath)]
+        self.si, self.ei = torchaudio.info(os.path.join(self.asset_dirpath, "sinewave.wav"))
+        self.si.precision = 16
+        self.E = torchaudio.sox_effects.SoxEffectsChain()
+        self.E.append_effect_to_chain("rate", [self.si.rate])  # resample to 16000hz
+        self.E.append_effect_to_chain("channels", [self.si.channels])  # mono singal
+        self.E.append_effect_to_chain("trim", [0, "16000s"])  # first 16000 samples of audio
+
+    def __getitem__(self, index):
+        fn = self.data[index]
+        self.E.set_input_file(fn)
+        x, sr = self.E.sox_build_flow_effects()
+        return x
+
+    def __len__(self):
+        return len(self.data)
+
+class Test_LoadSave(unittest.TestCase):
```

Comment:
[Minor] Wrong name?

Reply:
I don't see what you mean here.  Could you be super explicit?  I'm sure it's just that I've overlooked at this code so many times before that I can't see which name is wrong.

Reply:
`Test_LoadSave` -> `Test_DataLoader` ... got it.